Components/Slider.js

```diff
@@ -0,0 +1,394 @@
+import React, {Component } from 'react';
```

Comment:
I'm going to guess this file is a slider from somewhere else 😛 . Just import it. If you made this yourself, let me know and I'll actually look it over.